music21/musicxml/m21ToXml.py

```diff
@@ -4800,13 +4870,9 @@ def articulationToXmlTechnical(self, articulationMark):
         >>> MEX.dump(mxOther)
         <other-technical>unda maris</other-technical>
 
-        Same with technical marks not yet supported.
-        TODO: support HammerOn, PullOff, Bend, Hole, Arrow.
-
         >>> h = articulations.HammerOn()
         >>> mxOther = MEX.articulationToXmlTechnical(h)
         >>> MEX.dump(mxOther)
-        <other-technical />
```

Comment:
```suggestion
        >>> MEX.dump(mxOther) is None
        True
```
Cosmetic thing I forgot to mention: so that it doesn't look in the Sphinx-built version of docs that something is missing or someone forgot to write the result.